build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
```

Comment:
This is a little bit of a misnomer: it's not really "floating test runtime dependencies", just the "latest" ones in test-runtime. 
Perhaps there's a way for us to creatively retrieve the latest version number of (say) Microsoft.NetCore from myget.org, then do a replace on the project.json file for the actual latest at the time.